compiler/rustc_target/src/spec/armv6_unknown_linux_dynmusleabihf.rs

```diff
@@ -0,0 +1,12 @@
+use crate::spec::Target;
+
+pub fn target() -> Target {
+    let mut base = super::arm_unknown_linux_musleabihf::target();
+
+    base.llvm_target = "armv6-unknown-linux-musleabihf".to_string();
+    base.options.vendor = "alpine".to_string();
```

Comment:
We should not set `vendor` here, as some other Linux distribution also using musl, e.g. Gentoo Linux or OpenWRT

Reply:
Yes, this was an error in the original patch.  Fixed now!